courses/models.py

```diff
@@ -14,9 +14,9 @@
 
 
 class Course(models.Model):
+    instructor = models.ForeignKey(User, on_delete=models.CASCADE, related_name="courses_taught")
```

Comment:
there could be multiple instructors
also I changed the models a bit (moved them to a folder), can you please update it? Thank you!

Reply:
Ok 👍🏾